ui/redux/actions/file.js

```diff
@@ -129,7 +114,7 @@ export const doFileGetForUri = (uri: string, onSuccess?: (GetResponse) => any) =
 
   dispatch({ type: ACTIONS.FETCH_FILE_INFO_STARTED, data: { outpoint } });
 
-  Lbry.get({ uri })
+  Lbry.get({ uri, stripeEnvironment })
     .then((streamInfo: GetResponse) => {
```

Comment:
In videojs, the param is defined as `environment` while it is defined as `stripeEnvironement` here. Both accepted?

```
          const protectedLivestreamResponse = await Lbry.get({
            uri: activeLivestreamForChannel.claimUri,
            base_streaming_url: activeLivestreamForChannel.url,
            environment: stripeEnvironment,
          });
```